release/models/keychain/openconfig-keychain.yang

```diff
@@ -194,7 +200,7 @@ module openconfig-keychain {
 
     leaf key-id {
       type union {
-        type oc-yang:hex-string {
+        type oc-yang:hex-string-prefixed {
```

Comment:
Isn't this now going to pose an issue for the `length 1..64` restriction?

`0x` is now a mandatory prefix and counts towards the string length

Reply:
updated

Reply:
Thx - I think the min length would probably need adjustment in that 3 is now the bare minimum (Just `0x` is insufficent to represent a hex-string unless we assume `0x0`).  The pattern regex will cover the type check irregardless.

It seems each usage can define a separate upper bounds but the lower bounds should always have a consistent minimum.

The typedef could carry something like the following:

`range 3..max;`

Whereas each use could override but maintain the min

`range 3..66;`
`range 3..254;`

Reply:
Thanks for the catch in the min and good idea to add the other length constraints.  I've updated.